eras/babbage/formal-spec/utxo.tex

```diff
@@ -173,6 +173,8 @@ \section{UTxO}
   \label{fig:rules:utxo-state-upd}
 \end{figure}
 
+In the UTXO rule, we switch from a manual estimation of the size consumed by $\UTxO$ entries to an estimation using the serialization. However, since the $\TxIn$ used as a key in the $\UTxO$ map is not part of the serialization, we need to account for it manually. By itself it is $40$ bytes, but we add another $128$ bytes of overhead for the in-memory representation of Haskell data.
```

Comment:
It should add up to 160, right? :wink: 
```suggestion
In the UTXO rule, we switch from a manual estimation of the size consumed by $\UTxO$ entries to an estimation using the serialization. However, since the $\TxIn$ used as a key in the $\UTxO$ map is not part of the serialization, we need to account for it manually. By itself it is $40$ bytes, but we add another $120$ bytes of overhead for the in-memory representation of Haskell data.
```

Reply:
Ah, I only changed it in one place, thanks!